src/app/templates/server/middleware/logger.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
+// @slajax TODO: re-write this logger so it scales
 const winston = require('winston');
 
-module.exports = function(options = {}) {
-  return function(error, req, res, next) {
+module.exports = function (options = {}) {
+  return function (error, req, res, next) {
     if (error) {
-      const message = `${error.code ? `(${error.code}) ` : '' }Route: ${req.url} - ${error.message}`;
-      
+      const message = `${error.code ? `(${error.code}) ` : ''} Route: ${req.url} - ${error.message}`;
+
       if (error.code === 404) {
-        winston.info(message);
-      }
-      else {
-        winston.error(message);
-        winston.info(error.stack);
+        if (process.env.NODE_ENV !== 'testing') {
+          winston.info(message);
+        }
+      } else {
+        if (process.env.NODE_ENV !== 'testing') {
+          winston.error(message);
+          winston.info(error.stack);
+        }
       }
```

Comment:
Will refactor this with an improved default solution for logging. Winston is great, but I feel like we can do better then this and even make it configurable for adapters like logstash.